### PR TITLE
Updating releases.json for vs-mac-support version

### PR DESCRIPTION
**Pull request by kalaskarsanket**
Updating releases.json for vs-mac-support version

### CODE REVIEW

File: release-notes/6.0/releases.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
         "vs-version": "17.0.0",
         "vs-mac-version": "8.10",
         "vs-support": "Visual Studio 2022 (v17.0)",
-        "vs-mac-support": "Visual Studio 2019 for Mac (v8.10)",
+        "vs-mac-support": "Visual Studio 2022 (v17.0)",
```
Comment (leecow) [inline release-notes/6.0/releases.json:128]: This needs to be "Visual Studio 2022 for Mac (v17.0 latest preview)".

File: release-notes/6.0/releases.json [hunk 2]
```diff
@@ -236,7 +236,7 @@
           "vs-version": "17.0.0",
           "vs-mac-version": "8.10",
           "vs-support": "Visual Studio 2022 (v17.0)",
-          "vs-mac-support": "Visual Studio 2019 for Mac (v8.10)",
+          "vs-mac-support": "Visual Studio 2022 (v17.0)",
```
Comment (leecow) [inline release-notes/6.0/releases.json:239]: same as above

**Review by leecow — CHANGES_REQUESTED** [review]

File: release-notes/6.0/releases.json [hunk 3]
```diff
@@ -125,7 +125,7 @@
         "vs-version": "17.0.0",
         "vs-mac-version": "8.10",
```
Comment (leecow) [inline release-notes/6.0/releases.json]: 17.0

File: release-notes/6.0/releases.json [hunk 4]
```diff
@@ -236,7 +236,7 @@
           "vs-version": "17.0.0",
           "vs-mac-version": "8.10",
```
Comment (leecow) [inline release-notes/6.0/releases.json]: 17.0

File: release-notes/6.0/releases.json (hunk 2 above)
Comment (kalaskarsanket) [reply release-notes/6.0/releases.json:239]: resolving in new commit.

File: release-notes/6.0/releases.json (hunk 1 above)
Comment (kalaskarsanket) [reply release-notes/6.0/releases.json:128]: resolving in new commit.

**Review by leecow — APPROVED** [review]

**Pull request merged**